acceptance-tests/package.json

```diff
@@ -8,6 +8,6 @@
   },
   "devDependencies": {
     "@cucumber/cucumber": "^9.1.0",
-    "puppeteer": "^20.7.2"
+    "puppeteer": "^20.3.0"
```

Comment:
Any reason we are downgrading this ?

Reply:
I can only assume this was done as part of Andrew's fixes so suggest you speak to him about the reasons - He was looking into why the Build was failing and got it working.

If you wish it to be re-instated please let me know and I will be happy to comply.